openid_connect.js

```diff
@@ -7,6 +7,36 @@ var newSession = false; // Used by oidcAuth() and validateIdToken()
 
 export default {auth, codeExchange, validateIdToken, logout};
 
+
+function authZUriHandler(r) {
```

Comment:
For discussion, should this function appear below the exported ones? I like the idea of being able to follow the logic by looking at this file. So when you initially see a helper function, it might be confusing?

Reply:
Its now at the bottom of the js file.